### PR TITLE
Manual check for Process 2021

### PR DESCRIPTION
**Pull request by jennyliang220**

### CODE REVIEW

File: lib/profiles/TR/CRD.js [hunk 1]
```diff
@@ -8,9 +8,13 @@ exports.config = {
 };
 
 const base = require('../base');
-const rules = base.removeRules(
+let rules = base.removeRules(
     require('./CR').rules,
     'sotd.candidate-review-end'
 );
 
+rules = base.insertAfter(rules, 'sotd.supersedable', [
```
Comment (deniak) [inline lib/profiles/TR/CRD.js:16]: This is quite messy.
* TR.js has `draft-stability`
* CR.js extends TR.js but removes `draft-stability` from the rules list

and now, CRD.js extends CR.js but re-adds `draft-stability`. This requires some clean up.

File: lib/profiles/TR/CRYD.js [hunk 2]
```diff
@@ -11,5 +11,8 @@ let { rules } = require('./CRY');
 const base = require('../base');
 
 rules = base.removeRules(rules, 'sotd.candidate-review-end');
+rules = base.insertAfter(rules, 'sotd.supersedable', [
```
Comment (deniak) [inline lib/profiles/TR/CRYD.js:14]: Same comment as above

File: lib/profiles/TR/DNOTE.js [hunk 3]
```diff
@@ -8,4 +8,9 @@ config.styleSheet = 'W3C-DNOTE';
 
 exports.config = config;
 
-exports.rules = note.rules;
+const base = require('../base');
+const rules = base.insertAfter(note.rules, 'sotd.supersedable', [
```
Comment (deniak) [inline lib/profiles/TR/DNOTE.js:12]: Same comment as above

File: lib/profiles/TR/DRY.js [hunk 4]
```diff
@@ -6,6 +6,11 @@ exports.config = {
     track: 'Registry',
 };
 
-const { rules } = require('./RY');
+let { rules } = require('./RY');
+const base = require('../base');
+
+rules = base.insertAfter(rules, 'sotd.supersedable', [
```
Comment (deniak) [inline lib/profiles/TR/DRY.js:12]: Same comment as above

**Review by deniak — CHANGES_REQUESTED** [review]
The profiles are quite messy. Please make some clean up so it's easier to follow what each profile actually checks.

**Comment by jennyliang220** [comment]
Will address the comments in another PR. See issue https://github.com/w3c/specberus/issues/1442

**Pull request merged**